frontend/common/jest.config.js

```diff
@@ -16,6 +16,10 @@ module.exports = {
     },
   },
   roots: ["src", "tests"],
+  // https://alexjover.com/blog/enhance-jest-configuration-with-module-aliases/
+  moduleNameMapper: {
+    "^.+\\.(css|less)$": "<rootDir>/CSSStub.js",
```

Comment:
Not sure if there's already a convention, but camel-casing acronyms in filenames feels a little easier to parse visually:

```suggestion
    "^.+\\.(css|less)$": "<rootDir>/CssStub.js",
```